### PR TITLE
Added governance members and motions

### PR DESCRIPTION
**Pull request by mysty0**
Added new page for displaying basic governance info with members list and current motions

image
image

### CODE REVIEW

File: app/routes/governance.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { useTranslation } from "react-i18next";
+import { NavLink, Outlet } from "@remix-run/react";
+
+// Add this function before your component
```
Comment (fedosov) [inline app/routes/governance.tsx:4]: What does that mean?

Comment (mysty0) [reply app/routes/governance.tsx]: It some residual from the cursor that I didn't spot. Cleaned it up in the latest commit

File: app/routes/governance.members.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import { useApi } from "../components/Api";
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Card, Elevation, Spinner } from "@blueprintjs/core";
+import type { AccountId } from "@polkadot/types/interfaces";
+import type { DeriveElectionsInfo, DeriveCouncilVotes } from '@polkadot/api-derive/types';
```
Comment (fedosov) [inline app/routes/governance.members.tsx:6]: Please fix this and other places to use consistent double quotes. Prettier should help (we have a configuration for the project).

File: app/routes/governance.members.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    parent?: string;
+    isGood: boolean;
+    isBad: boolean;
+  };
+  votes: number;
+  balance: string;
+}
+
+const transformVotes = {
+  transform: (entries: DeriveCouncilVotes): Record<string, AccountId[]> =>
```
Comment (fedosov) [inline app/routes/governance.members.tsx:24]: Why this nesting? We can simplify this. Let's keep the code as simple as possible.

File: app/routes/governance.members.tsx (hunk 2 above)
Comment (mysty0) [reply app/routes/governance.members.tsx]: I applied prettier to all files in this pr

File: app/routes/governance.members.tsx (hunk 3 above)
Comment (mysty0) [reply app/routes/governance.members.tsx]: Fixed it

**Pull request merged**